specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,104 @@
+# Date/Time Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer, a decimal value, or a value represented in scientific notation as defined in this section.
+
+## Attribute ID
```

Comment:
I would consider putting Numeric Format, Numeric Precision and Examples above this line (maybe under sub headings (H3 or lower) so we don't mix up the descriptive stuff with the spec items that start with this Attribute ID header. Feels like Numeric Format being the first section and then precision and then examples make sense to me).

I think there is a normative sentence in the Precision section that we can probably move into the above normative section (line 5)

Reply:
I rearranged the sections.